src/jsonrpccpp/server/connectors/httpserver.h

```diff
@@ -49,6 +51,16 @@ namespace jsonrpc
              */
             HttpServer(int port, const std::string& sslcert = "", const std::string& sslkey = "", int threads = 50);
 
+            /**
+             * Sets the address at which the server will bind when started.
+             * By default, the server listens at all interfaces.  If this
+             * method is called with a non-empty string before starting the
+             * server, then it will only bind to the specified address.
+             */
+            void SetBindAddress(const std::string& addr) {
+              bind_address = addr;
```

Comment:
Please move the implementation to the cpp file.

Reply:
Done.